src/lms-harmonizer/edfi_lms_harmonizer/exceptions_reports.py

```diff
@@ -68,3 +99,15 @@ def create_exception_reports(adapter: Adapter, output_directory: str) -> None:
     logger.info("Writing the Users exception report")
     users = pd.read_sql(QUERY_FOR_USERS, adapter.engine)
     users.to_csv(_get_file_path(output_directory, USERS), index=False)
+
+    logger.info(
+        "Writing the Assignment Category and Submission Status missing descriptors report"
+    )
+    assignment_cat = pd.read_sql(QUERY_FOR_ASSIGNMENT_CAT_DESCRIPTORS, adapter.engine)
+    submission_status = pd.read_sql(
+        QUERY_FOR_SUBMISSION_STATUS_DESCRIPTORS, adapter.engine
+    )
+    descriptors_report = pd.concat([assignment_cat, submission_status], axis=1)
+    descriptors_report.to_csv(
+        _get_file_path(output_directory, DESCRIPTORS), index=False
+    )
```

Comment:
Here's a sample output right now:

```
AssignmentCategory,SubmissionStatus
Assessment,Created
Assignment,Late
,on-time
,Returned
```

Translated to a markdown table for reading:

|AssignmentCategory | SubmissionStatus |
| -- |-- |
|Assessment|Created|
|Assignment|Late|
||on-time|
||Returned|

Reading it in Excel, like the markdown above, is pretty straightforward. But it does look weird in CSV.

What do you think about changing to the following output format? Also note the addition of the expected namespace

```
Descriptor, Namespace, Missing Value
AssignmentCategoryDescriptor, uri://ed-fi.org/edfilms/AssignmentCategoryDescriptor/BestLMS, Assessment
AssignmentCategoryDescriptor, uri://ed-fi.org/edfilms/AssignmentCategoryDescriptor/BestLMS, Assignment
SubmissionStatusDescriptor, uri://ed-fi.org/edfilms/SubmissionStatusDescriptor/BestLMS, Created
SubmissionStatusDescriptor, uri://ed-fi.org/edfilms/SubmissionStatusDescriptor/BestLMS, Late
SubmissionStatusDescriptor, uri://ed-fi.org/edfilms/SubmissionStatusDescriptor/BestLMS, on-time
SubmissionStatusDescriptor, uri://ed-fi.org/edfilms/SubmissionStatusDescriptor/BestLMS, Returned
```

Reply:
Yes, will commit a new version in a few minutes